README.md

```diff
@@ -19,18 +19,21 @@ To run the **HR Management** Web App, ensure your system meets the following req
 Follow the steps below to setup the project locally:
 
 * clone the repository: 
-git clone https://github.com/harry-olabs/avi-hub.git
+git clone 
+
+`https://github.com/harry-olabs/avi-hub.git`
+
```

Comment:
This should be together on a new line. Something like this:

```
git clone https://github.com/harry-olabs/avi-hub.git
```

This is a code block format ☝️. If you hover over it, GitHub allows you to click copy button to copy the command.